TrackingTools/PatternTools/src/classes_def.xml

```diff
@@ -25,7 +25,7 @@
     <field name="transientMap_" transient="true"/>
   </class>
 
-  <class name="edm::Wrapper<edm::AssociationMap<edm::OneToOne<std::vector<Trajectory>,std::vector<reco::Track>,unsigned short> > >" persistent="false" />
+  <class name="edm::Wrapper<edm::AssociationMap<edm::OneToOne<std::vector<Trajectory>,std::vector<reco::Track>,unsigned short> > >" />
```

Comment:
Is the intention to write this data product into a file?

Reply:
yes, with TICLv5 we need to save it for rereco (it's used to compute the path length in the `TICLCandidateProducer.cc`)

Reply:
Then `Trajectory` (and all it depends on) would need to be moved to some `DataFormats` package. On the other hand, IIRC `Trajectory` was never intended to be stored in a file, so perhaps this suggests some other solution should be sought?

Reply:
I've undone this change